bokehjs/src/coffee/models/tickers/months_ticker.coffee

```diff
@@ -56,7 +54,7 @@ export class MonthsTicker extends SingleIntervalTicker
         month_date.setUTCMonth(month)
         return month_date)
 
-    month_dates = _.flatten(months_of_year(date) for date in year_dates)
+    month_dates = [].concat((months_of_year(date) for date in year_dates)...)
```

Comment:
`flatten()` has additional meaning assigned to it. It not only concatenates arrays, but also allows scalar values and recursive behavior. We don't need this here. Perhaps I could add a convenience function `concat()` for this purpose. Note that `[].concat(arrays...)` is an idiom.